packages/node/package.json

```diff
@@ -12,20 +12,22 @@
   "scripts": {
     "build": "node build",
     "test": "jest --env node --verbose --bail --runInBand",
-    "test-unit": "pnpm test test/prepare-cache.test.ts test/utils.test.ts test/dev.test.ts",
+    "test-unit": "pnpm test test/prepare-cache.test.ts test/utils.test.ts test/dev.test.ts test/edge-matcher.test.ts",
     "test-e2e": "pnpm test test/integration-*.test.js"
   },
   "files": [
     "dist"
   ],
   "jest": {
     "preset": "ts-jest",
-    "testEnvironment": "node",
```

Comment:
Is the test environment not needed?

Reply:
`testEnvironment` defaults to `"node"`, so we don't *need* this. https://jestjs.io/docs/configuration#testenvironment-string